src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -488,10 +489,11 @@ def minimize_corpus(self, target_path, arguments, input_dirs, output_dir,
     if result.return_code != 0:
       raise MergeError('Merging new testcases failed: ' + result.output)
 
-    merge_stats = stats.parse_stats_from_merge_log(result.output.splitlines())
+    merge_output = utils.decode_to_unicode(result.output)
```

Comment:
I keep thinking if we should decode_to_unicode the output in all cases (in run_and_wait ?), so that we don't forget to explicitly do it, thoughts ?

Reply:
Not sure if that makes sense more generally, since processes can return non-unicode/text content. For libFuzzer/engines, it makes sense because that's what we expect.

Reply:
I think it may make sense for process_handler.run_process, since that's mostly used for running blackbox targets which should output text content.